### DISCUSSION TITLE
Thank you!: Community growth & development update

### DISCUSSION THREAD

**Discussion by jamesread** [OliveTin/OliveTin#23] [Announcements]
I also just wanted to say a big thank you - 30 people in the discord, and 220 Github :star:s now. That's great to see the community growing. 

When I have free time, I'm working away busily on a big update that will include things like task arguments, improved automated tests, entities, and better update notifications - so lots of things happening behind the scenes for the next release.

Thank you all for joining and helping the project grow :-)

**Comment by ashleycawley** [comment]
🎉 Thank you for doing the hard work on the coding!

**Comment by mangaxx** [comment]
I found this product fantastic for its semplicity and usability

Is it possible to ask for any new implementation about that?

**Reply by jamesread** [reply]
Hey @mangaxx Are you talking about asking for new features? Sure, feel free to raise an issue or jump in the Discord server to chat about what you had in mind. You can find a link to the Discord server at http://olivetin.app
